shenyu-common/src/main/java/org/apache/shenyu/common/utils/UriUtils.java

```diff
@@ -42,6 +42,18 @@ public static URI createUri(final String uri) {
         return null;
     }
 
+    /**
+     * create URI {@link URI}.
+     *
+     * @param scheme    scheme eg:http
+     * @param authority registry or server eg: 127.0.0.1:8080
+     * @param path      path eg:/ fallback
+     * @return created {@link URI} from uri
+     */
+    public static URI createUri(final String scheme, final String authority, final String path) {
```

Comment:
you can use springframework `UriComponentBuilder`

Reply:
ok, i'll fix it

Reply:
As this class `UriUtils` is in `shenyu-common`, can i add `spring-web` dependency into `shenyu-common` module?

Reply:
no,

Reply:
ok, i'll not fix it now

Reply:
@moremind  can we skip this change?